<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plurinotes</class>
 <widget class="QMainWindow" name="Plurinotes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="noteList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>211</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="archiveList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="taskList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>211</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <widget class="QMenu" name="menuNouveau">
     <property name="title">
      <string>Nouveau</string>
     </property>
     <addaction name="actionArticle"/>
     <addaction name="actionTache"/>
     <addaction name="actionMedia"/>
    </widget>
    <addaction name="menuNouveau"/>
    <addaction name="separator"/>
    <addaction name="actionOuvrir"/>
    <addaction name="actionSauver"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionAnnuler_Ctrl_Z"/>
    <addaction name="actionR_tablir_Ctrl_Y"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionArticle">
   <property name="text">
    <string>Article</string>
   </property>
  </action>
  <action name="actionTache">
   <property name="text">
    <string>Tache</string>
   </property>
  </action>
  <action name="actionMedia">
   <property name="text">
    <string>Media</string>
   </property>
  </action>
  <action name="actionAnnuler_Ctrl_Z">
   <property name="text">
    <string>Annuler (Ctrl+Z)</string>
   </property>
  </action>
  <action name="actionR_tablir_Ctrl_Y">
   <property name="text">
    <string>RÃ©tablir (Ctrl+Y)</string>
   </property>
  </action>
  <action name="actionSauver">
   <property name="text">
    <string>Sauver</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionArticle</sender>
   <signal>triggered()</signal>
   <receiver>Plurinotes</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
